activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/configuration/RuntimeBundleProperties.java

```diff
@@ -90,7 +90,7 @@ public void setEventsProperties(RuntimeBundleEventsProperties eventsProperties)
 
     public static class RuntimeBundleEventsProperties {
 
-        private boolean integrationAuditEventsEnabled;
+        private boolean integrationAuditEventsEnabled = true;
```

Comment:
cannot `true` be applied as default in the application yaml file instead?

Reply:
I chose this approach because is what @erdemedeiros suggested in the Jira issue comments (https://issues.alfresco.com/jira/browse/AAE-763?focusedCommentId=727465&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-727465), but I have no problem in setting it in the YAML instead.

Reply:
if we remove `true` from here, `false` becomes the default and the application yaml file will override the default. Reading the old conversation we had, yes the default should be `true` so i resolve this comment

Reply:
Yes, I think it's fine to have `true` as default value here.